f2/src/forms/EvidenceInfoForm.js

```diff
@@ -108,9 +109,9 @@ export const EvidenceInfoForm = props => {
             <Container>
               {files.map((f, index) => (
                 <React.Fragment key={index}>
-                  <InfoCard mb={4}>
+                  <InfoCard as="article" mb={4}>
                     <Stack spacing={4}>
-                      <H3 fontSize="2xl">{f.name}</H3>
+                      <H1 fontSize="2xl">{f.name}</H1>
```

Comment:
We can't have another H1 on this page for a11y reasons - can we style the H3 (or an H2) to be the size you want?

Reply:
Good catch, I was under the impression that articles needed an h1 as they are self contained.